scripts/verify.js

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env node
+
+const util = require("util");
+const path = require("path");
+const exec = util.promisify(require("child_process").exec);
+const debugMode = process.env.DEBUG_CONTRACT_VERIFICATION || false;
+const verifyCMD = `./node_modules/.bin/truffle run verify ${
+  debugMode ? "--debug --network" : "--network"
+}`;
+
+const contractsToVerify = [
+  "DaoRegistry",
+  "BankExtension",
+  "BankFactory",
+  "NFTExtension",
+  "NFTCollectionFactory",
+  "DaoFactory",
+  "VotingContract",
+  "ConfigurationContract",
+  "RagequitContract",
+  "ManagingContract",
+  "FinancingContract",
+  "OnboardingContract",
+  "GuildKickContract",
+  "DaoRegistryAdapterContract",
+  "BankAdapterContract",
+  "NFTAdapterContract",
+  "CouponOnboardingContract",
+  "TributeContract",
+  "DistributeContract",
+  "TributeNFTContract",
+  "SnapshotProposalContract",
+  "KickBadReporterAdapter",
+  "OffchainVotingContract",
+  "Multicall",
```

Comment:
Multicall should not be verified. there is one already verified on rinkeby and mainnet

Reply:
could we use ContractUtil.contracts to avoid having this list in multiple places ?

Reply:
it might be possible, just need to think about how to create an allow list to verify only the ones in this list, otherwise every contract declared in the CotnractUtil.contracts would be verified.